packages/core/src/render3/component_ref.ts

```diff
@@ -351,10 +347,10 @@ export class ComponentFactory<T> extends AbstractComponentFactory<T> {
       // issues would allow us to drop this.
       enterView(rootLView);
 
-      let component: T;
       let componentView: LView | null = null;
 
       try {
+        // TODO(pk): this is a good refactoring since it shows how to instantiate a TNOde with a set of known directive defs
```

Comment:
This comment is a bit unclear: which refactoring do we refer to?
```suggestion
        // TODO(pk): this is a good refactoring since it shows how to instantiate a TNode with a set of known directive defs
```